Genez-io/genezio-tests
# create list delete integration test

Test flow:

- create a new project using `genezio init` and `genezio addClass` (content of class is the same as `helo-world`)
- depoy the project using `genezio deploy`
- use `genezio ls` to check if the newly created project is listed
- use `genezio ls <project_name>` to get the ID of the project
- delete the project using `genezio delete <ID>`
- check again with `genezio ls` that the project is not listed
